### ISSUE TITLE
Is it possible to see errors from a container in community.docker.docker_compose_v2?

### ISSUE THREAD

**Issue by mihalt** [ansible-collections/community.docker#979]
I would like not to ssh to my server with `docker ps -a` and after `docker logs container_name` each time when error occures in any container. And see them immidiately in ansible output. Is it possible?

**Comment by felixfontein** [comment]
@mihalt can you please provide more details, for example how you would do this without Ansible?

**Comment by mihalt** [comment]
> I'm not sure what exactly you need. A module which allows you to run `docker logs <containername>` for one / all containers for a service and/or project?

For example I would like to have an option of you collection to choose one of this parameters of showing logs:
1. Logs of first failled container with problems (by default)
2. Show all failed containers log
3. Show last failed container logs

Maybe something like this.

**Comment by felixfontein** [comment]
So which command do you run without Ansible to achieve this? The docker_compose_v2 module allows you to run commands like `docker compose up`, `docker compose down`, etc.

**Comment by mihalt** [comment]
> So which command do you run without Ansible to achieve this? The docker_compose_v2 module allows you to run commands like `docker compose up`, `docker compose down`, etc.

hmm, maybe `docker compose logs`, could be ok. But output is too big

**Comment by SiyaDadheech** [comment]
yes, to see container logs the command is: `docker container logs **<container-id>** or **<container-name>** `
to go inside the container and see the log the command is:` docker container exec -it **<container-name>**  bash` , to inspect files, logs internal to the container, configuration, etc.
docker compose logs to see the stdout/stderr output from container
